books/nlp/src/llms/agents/README.md

```diff
@@ -1 +1,119 @@
+<!-- markdownlint-disable-file MD033 -->
+
 # Agents
+
+{{ #aipr_header }}
+
+Agents are LLMs equipped with [tools](./tool_use.md) and memory to interact
+with the environment and complete specific, user-defined objectives.
+They go about this by following workflows which direct them in
+(i) [planning](./planning.md) for what steps and tools are needed,
+(ii) executing an action, and (iii) [reflecting](./reflection.md) on feedback
+from their action, looping through these steps
+when the initial plan calls for multiple actions or reflection
+deems more actions to be needed to complete the objective.
```

Comment:
Bit of a run-on sentence here. Suggest editing to: "...looping through these steps when the initial plan requires multiple actions, or when reflection suggests additional actions are needed to achieve the objective."